1733Sprint/1-Kenny/Demo.java

```diff
@@ -0,0 +1,49 @@
+import vehicles.*;
```

Comment:
It's common practice to not use a * for your imports: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad